release_notes.py

```diff
@@ -154,10 +154,15 @@ def release_notes_section(self, store, title_prefix, current_server_version):
                     cnt = self.load_data(raw_content)
 
                     from_version = cnt.get("fromversion") or cnt.get("fromVersion")
+                    to_version = cnt.get("toversion") or cnt.get("toVersion")
                     if from_version is not None and server_version_compare(current_server_version, from_version) < 0:
-                        print("Skipped because of version differences")
+                        print(f'Skipped because from version: {from_version}'
+                              f' is greater than current server version: {current_server_version}')
+                        continue
+                    if to_version is not None and server_version_compare(to_version, current_server_version) < 0:
```

Comment:
currently, we run this script with server version 5.0.0
changes made for 4.5 will not appear in the release notes